osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -122,8 +130,14 @@ private double computeAimValue(ScoreInfo score, OsuDifficultyAttributes attribut
 
             if (attributes.SliderCount > 0)
             {
-                double estimateSliderEndsDropped = Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders);
-                double sliderNerfFactor = (1 - attributes.SliderFactor) * Math.Pow(1 - estimateSliderEndsDropped / estimateDifficultSliders, 3) + attributes.SliderFactor;
+                double estimateSliderEndsDropped;
+                if (score.IsLegacyScore)
+                    estimateSliderEndsDropped = Math.Clamp(Math.Min(countOk + countMeh + countMiss, attributes.MaxCombo - scoreMaxCombo), 0, estimateDifficultSliders);
+                else
+                    estimateSliderEndsDropped = countSliderEndsDropped;
```

Comment:
1. What about this is incorrect?

2. I’m hesitant to do that since I’d imagine values using `estimateSliderEndsDropped` are balanced around sliderend drops, not sliderend drops + other stuff

Reply:
what if you hit all sliderends but missed all reverses?
you will still get full slideraim value?

Reply:
> 1. What about this is incorrect?
> 2. I’m hesitant to do that since I’d imagine values using `estimateSliderEndsDropped` are balanced around sliderend drops, not sliderend drops + other stuff

also, it's not balanced around sliderend drops, because in case where you not dropped sliderend but sliderbroke - you almost instantly loosing almost all slider pp

Reply:
also change it to `Math.Min(countSliderEndsDropped + countLargeTickMiss, estimateDifficultSliders);`
because otherwise you can get negative pp by missing to many sliderends